source/language/classical.rst

```diff
@@ -1,17 +1,18 @@
 Classical instructions
 ======================
 
-We envision two levels of classical control that we call low-level
-instructions and high-level kernel functions. Simple, fast instructions
-control the flow of the program and allow basic computations on
-lower-level parallel control processors. These instructions are likely
+We envision two levels of classical control: simple, low-level instructions embedded as
+part of a quantum circuit and high-level external functions which perform more complex
+classical computations. The low-level functions are fast, apply control flow and allow basic
```

Comment:
```suggestion
classical computations. The low-level functions allow basic
```
I think we can simplify this further. In particular, OpenQASM doesn't have branching _instructions_, but it does have branching _syntax_.

Reply:
Addressed in https://github.com/Qiskit/openqasm/pull/204/commits/40df87657528bc9c5aa65355cc79175c01763fc7.